website/client/components/userLink.vue

```diff
@@ -11,10 +11,6 @@
 <style scoped lang="scss">
   @import '~client/assets/scss/colors.scss';
 
-  a {
-    color: $gray-50;
```

Comment:
@negue What was this for? It's thrown off the styling of non-contributors in chat--gonna work on a hotfix to put it back, but curious what this was intended to do

Reply:
I removed that to be able to show the contribution level (colors) but I guess that color was needed for all other links, I'll push that back in on the infinite scrolling PR

Reply:
OK! We sent out a hotfix in 3eb164a and d7f6c3b, so it's working now but ideally we shouldn't be using `!important` for the CSS there, heh